### ISSUE TITLE
`aria-owns` has limited support in some user agents

### ISSUE THREAD

**Issue by dd8** [auto-wcag/auto-wcag#2275]
## Accessibility support

`aria-owns` has limited support in iOS 16 and macOS 13. These versions reach end-of-life in October 2025 when new versions of iOS and macOS are released (Apple support the current version, and previous 2 versions)

## ACT Rules

- https://www.w3.org/WAI/standards-guidelines/act/rules/bc4a75/
- https://www.w3.org/WAI/standards-guidelines/act/rules/ff89c9/

## Test cases

Test cases from https://a11ysupport.io/tech/aria/aria-owns_attribute

### Test case 1

```html
<ul id="target">
    <li aria-owns="child">Fruit</li>
    <li>Vegetables</li>
</ul>

<ul id="child">
    <li>Apples</li>
    <li>Bananas</li>
</ul>
```

[Open this test case](https://a11ysupport.io/tests/html/aria/aria-owns-single.html)

### Test case 2

```html
<ul id="target">
    <li aria-owns="child1 child2">Fruit</li>
    <li>Vegetables</li>
</ul>

<ul id="child1">
    <li>Apples</li>
    <li>Bananas</li>
</ul>

<ul id="child2">
    <li>Blackberries</li>
    <li>Blueberries</li>
</ul>
```

[Open this test case](https://a11ysupport.io/tests/html/aria/aria-owns-multiple.html)

#### Preconditions

Before running the test, check your settings. In many cases default settings aren't strictly required and will not impact the results. Use common sense here. If non-default settings are necessary for you, record the settings used in the reported results.

- Ensure browsers are in the default settings, turn off unnecessary extensions
- Ensure ATs are in the default settings
- Ensure operating system are in default settings
- Ensure software uses the correct version (usually the latest version)

#### Test instructions

- Tested in the chosen combination of screen reader and browser (e.g. VoiceOver with Safari, or JAWS with Chrome)
- No changes to defaults are required
- Check that the speech rendering of the test case matches the "How example should be rendered" section following the test case

### External links

[`aria-owns` is now supported](https://a11ysupport.io/tech/aria/aria-owns_attribute) in all modern browsers, but aria-owns may not be exposed to users of macOS and iOS using VoiceOver prior to iOS 17.3 and macOS 14.3.

## Test results

See https://a11ysupport.io/tech/aria/aria-owns_attribute
